stable/schema-registry-ui/Chart.yaml

```diff
@@ -10,7 +10,7 @@ keywords:
   - Kafka
   - avro
   - schema-registry
-version: 0.1.0
+version: 0.1.1
```

Comment:
I would bump to 0.2.0 since this is not a bug fix

Reply:
I agree, thanks for the spot. 

Updated on 11fbfac1390746b67fe1a317f500aec43f6e1938